src/components/DayPickerRangeController.jsx

```diff
@@ -643,38 +651,55 @@ export default class DayPickerRangeController extends React.Component {
   }
 
   deleteModifier(updatedDays, day, modifier) {
-    const { numberOfMonths, enableOutsideDays } = this.props;
-    const { currentMonth, visibleDays } = this.state;
+    const { numberOfMonths: numberOfVisibleMonths, enableOutsideDays, orientation } = this.props;
+    const { currentMonth: firstVisibleMonth, visibleDays } = this.state;
+
+    let currentMonth = firstVisibleMonth;
+    let numberOfMonths = numberOfVisibleMonths;
+    if (orientation !== VERTICAL_SCROLLABLE) {
```

Comment:
This `if` logic (ie, getting `currentMonth` and `numberOfMonths`) is repeated in DayPickerRangeController, and in SingleDatePicker. Could we move it to a shared function instead?